backend/services/mgmt/v1alpha1/user-account-service/billing.go

```diff
@@ -150,6 +151,11 @@ func (s *Service) IsAccountStatusValid(
 	ctx context.Context,
 	req *connect.Request[mgmtv1alpha1.IsAccountStatusValidRequest],
 ) (*connect.Response[mgmtv1alpha1.IsAccountStatusValidResponse], error) {
+	accountId, err := s.verifyUserInAccount(ctx, req.Msg.GetAccountId())
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This isn't really necessary here because the `GetAccountStatus` handles it.

Reply:
updated